calculator/calculator.py

```diff
@@ -0,0 +1,25 @@
+def add(a, b):
+    return int(a) + int(b)
+def subtract(a, b):
+    return int(a) - int(b)
+def multiply(a, b):
+    return int(a) * int(b)
+def divide(a, b):
+    return int(a) / int(b)
+
+print("Select Operation:\n1- Add,\n2- Subtract,\n3- Multiply,\n4- Divide.")
+option = int(input("Enter you choice from 1/2/3/4 :- "))
+num1 = input("Enter your first number :- ")
+num2 = input("Enter your second number :- ")
+if option == 1:
+        print("results : ", add(num1,num2))
+
```

Comment:
These blank lines are not needed. Since it is one logical code block. A good rule of thumb would be to leave blank line whenever the code flow changes.

For example: before/ after `for/while` loops, `if elif else` block after before return if the function is large enough.